user-docs/latest/content-manager/writing-content.md

```diff
@@ -0,0 +1,99 @@
+# Writing content
```

Comment:
I'm wondering if this part wouldn't be more relevant upper in the document. 

You talked a lot of content, entries, etc. and then, at the end of all the explanation on how the Content Manager works, we can read a mention of how writing content :D